okteto.yml

```diff
@@ -0,0 +1,45 @@
+name: node-compose-getting-started
+
+build:
+  api:
```

Comment:
I would remove `api` and `web` from here, the compose file already has a build section.
Also, you can remove the `image` field from the compose file